src/Models/Services/Utils.ts

```diff
@@ -628,11 +628,13 @@ export const getResourceUrl = (
         if (typeof resource === 'string') {
             // it means the option is manually entered using freeform
             if (resourceType) {
-                switch (resourceType) {
-                    case AzureResourceTypes.DigitalTwinInstance:
-                    case AzureResourceTypes.StorageAccount:
-                        return resource;
-                    case AzureResourceTypes.StorageBlobContainer: {
+                switch (resourceType.toLowerCase()) {
```

Comment:
I see we are adding a lot of ToLowerCase on exisitng code. Why do we need to do this now?

Reply:
as I mentioned in my other comment, since Resource Graph returns type all lower case while other management calls return as it is described as enum in our codebase I needed to do this lowercase checks, instead I can create a util which does lowercase comparsion.

Reply:
for example Microsoft.Authorization roleAssignments type resources returns as usual in our enum using that service provider endpoint in management api:
![image](https://user-images.githubusercontent.com/45217314/201764620-420c6be9-f45f-4048-875c-70108d15cec9.png)

whereas Resource Graph endpoint returns:
![image](https://user-images.githubusercontent.com/45217314/201764815-144a6dff-a884-4ceb-8947-20591e7b0994.png)